ui/src/singletons/store.ts

```diff
@@ -233,7 +244,39 @@ const useStore = create<AppStore>()(
             }
           }),
       },
-    }),
+    }), {
+
+      limit: 50,
+
+        handleSet: (handleSet) =>
+          throttle<typeof handleSet>(1500, (state) => {
+            console.info("handleSet called")
+            handleSet(state)
+          }),
+        
+      
+        partialize: (state) => {
+          const newNodes = state.nodes.map((node) => {
+            const n = {...node}
+            delete n.width
+            delete n.height
+            delete n.selected
+            delete n.draggable
+            delete n.dragging
+            delete n.positionAbsolute
+            return n
+          })
```

Comment:
:memo: Alternatively, you might be to exclude the unwanted keys when destructuring the `node`. For example like option B here: https://github.com/charkour/zundo?tab=readme-ov-file#exclude-fields-from-being-tracked-in-history